src/tools/pub/cmd/pub-uploader.md

```diff
@@ -3,10 +3,10 @@ title: dart pub uploader
 description: Use dart pub uploader to add or remove uploaders for your Dart package on the pub.dev site.
 ---
 
-{{site.alert.tip}}
-  Instead of specifying uploaders for each package you publish,
-  consider using a
-  [verified publisher](/tools/pub/verified-publishers).
+{{site.alert.warning}}
+  The `dart pub uploader` command is _deprecated_ and will be removed in Dart
+  2.17. Package owners can manage uploaders using the admin page on pub.dev:
```

Comment:
Should we add something like:
```suggestion
  2.17. And will stop working for earlier versions too. Package owners can manage uploaders using the admin page on pub.dev:
```

Reply:
I'm not sure there is any reason to stop older versions from using the API.
But I don't oppose to it, since it simplifies code on pub.dev, and that is good :D

Reply:
Also filed https://github.com/dart-lang/pub-dev/issues/5566